designsafe/static/scripts/projects/components/manage-field-recon/missions/manage-field-recon-missions.component.js

```diff
@@ -249,9 +248,7 @@ class ManageFieldReconMissionsCtrl {
         this.data.editMission.value.dateStart = new Date(
             this.data.editMission.value.dateStart
         );
-        this.data.editMission.value.dateEnd = new Date(
-            this.data.editMission.value.dateEnd
-        );
+        this.data.editMission.value.dateEnd = (this.data.editMission.value.dateEnd && this.data.editMission.value.dateEnd !== 'None') ? new Date(this.data.editMission.value.dateEnd) : '';
```

Comment:
Our linters would probably flag this line.  Can we break it into multiple lines, or make it simpler?

Reply:
Yeah, it's... long. The variables themselves are just too long! 😞 I'll throw it into an if else statement.